go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/tidelift/tidelift-sbom-info
 
-go 1.22
+go 1.23
```

Comment:
needed this at first for `maps.Keys`, but decided against it in the end. Here are the release notes: https://tip.golang.org/doc/go1.23